services/bettersocial/post/createPostV3.js

```diff
@@ -86,12 +86,16 @@ function getPollPostExpiredAt(pollsDuration, durationFeed) {
 }
 
 async function processPost({userId, isAnonimous, body, data, filteredTopics}) {
+  const expiredAt =
+    body.duration_feed !== 'never'
+      ? moment().utc().add(+body.duration_feed, 'day')
+      : moment().utc().add(100, 'years');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for setting the `expiredAt` variable seems to be correct. However, it's not a good practice to hardcode values like '100' for years directly in the code. It would be better to define a constant at the top of your file or in a separate constants file and use that instead. This will make your code more maintainable and easier to understand.

```diff
- : moment().utc().add(100, 'years');
+ const NEVER_EXPIRE_YEARS = 100;
+ : moment().utc().add(NEVER_EXPIRE_YEARS, 'years');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->